.goreleaser.yaml

```
@@ -0,0 +1,248 @@
+      - --label=org.opencontainers.image.version={{ .Env.VERSION }}
+      - --label=org.opencontainers.image.created={{ time "2006-01-02T15:04:05Z07:00" }}
+      - --label=org.opencontainers.image.revision={{ .FullCommit }}
+      - --label=org.opencontainers.image.licenses=Apache-2.0
+      - --platform=linux/arm/v7
+
+docker_manifests:
+  # NOTE: https://github.com/orgs/goreleaser/discussions/5822 ; https://github.com/goreleaser/goreleaser/issues/5824
+  - name_template: "{{ .Env.REGISTRY }}/{{ .Env.IMAGE_NAME }}:{{ .Env.VERSION }}"
+    image_templates:
+      - "{{ .Env.REGISTRY }}/{{ .Env.IMAGE_NAME }}:{{ .Env.VERSION }}-amd64"
+      - >- 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)